internal/server/v2/observation/calculation_util.go

```diff
@@ -18,17 +18,21 @@ import (
 	"fmt"
 	"go/ast"
 	"go/parser"
+	"go/token"
 	"reflect"
 	"strings"
 
 	pb "github.com/datacommonsorg/mixer/internal/proto"
 	pbv2 "github.com/datacommonsorg/mixer/internal/proto/v2"
+	"google.golang.org/protobuf/proto"
 )
 
 // The info of a node in the AST tree.
 type ASTNode struct {
 	StatVar string
 	Facet   *pb.Facet
+	// Map of entity -> facetId -> series.
+	CandidateSeries map[string]map[string][]*pb.PointStat
```

Comment:
CandidateObs? since this could be map points (not series)

Reply:
Done